.travis.yml

```diff
@@ -4,12 +4,18 @@ php: [5.3, 5.4, 5.5, 5.6, 7, hhvm]
 
 env:
   global:
+    - IOJS_VERSION=''
     - ZOMBIE_VERSION='@^3.0' # npm will install Zombie 4.x by default, even though it is not compatible with node...
 
 matrix:
   include:
     - php: 5.6
       env: ZOMBIE_VERSION='@^2.0'
+  include:
```

Comment:
you should not duplicate the ``include`` key in YAML but just put another element in the array